media/starboard/starboard_renderer.h

```diff
@@ -59,11 +59,37 @@ class MEDIA_EXPORT StarboardRenderer final : public Renderer,
     // TODO(b/380935131): Consider to implement `LatencyHint` for SbPlayer.
     NOTIMPLEMENTED();
   }
+  void SetPreservesPitch(bool preserves_pitch) final {
+    if (!preserves_pitch) {
```

Comment:
Let's just use LOG(INFO) here.  If LOG(WARNING) is preferred, consider using LOG_IF(WARNING, ...).
The same to the function below.

Reply:
There's also `NOTIMPLEMENTED()` / `NOTIMPLEMENTED_LOG_ONCE()` (https://source.chromium.org/chromium/chromium/src/+/main:base/notimplemented.h;l=15;bpv=0;bpt=1) as a suggestion.

Reply:
Both are only supported when DCHECK is on.

Reply:
Done.